servicebus/data-plane/azure-servicebus/src/main/java/com/microsoft/azure/servicebus/ClientSettings.java

```diff
@@ -0,0 +1,91 @@
+package com.microsoft.azure.servicebus;
```

Comment:
license header

Reply:
Will update the changes in one of next PR.